vllm/spec_decode/draft_model_runner.py

```diff
@@ -4,28 +4,22 @@
 
 from vllm.forward_context import set_forward_context
 from vllm.model_executor.layers.sampler import SamplerOutput
-from vllm.platforms import current_platform
 
-if current_platform.is_cuda_alike():
+try:
     try:
+        from vllm.attention.backends.flash_attn import FlashAttentionMetadata
+    except (ModuleNotFoundError, ImportError):
+        # vllm_flash_attn is not installed, try the ROCm FA metadata
         try:
-            from vllm.attention.backends.flash_attn import (
-                FlashAttentionMetadata)
-        except (ModuleNotFoundError, ImportError):
-            # vllm_flash_attn is not installed, try the ROCm FA metadata
             from vllm.attention.backends.rocm_flash_attn import (
                 ROCmFlashAttentionMetadata as FlashAttentionMetadata)
-    except (ModuleNotFoundError, ImportError, AssertionError) as err:
-        raise RuntimeError(
-            "Draft model speculative decoding currently only supports"
-            "CUDA and ROCm and HPU attention backend.") from err
-elif current_platform.is_hpu():
-    from vllm.attention.backends.hpu_attn import (HPUAttentionMetadata as
-                                                  FlashAttentionMetadata)
-else:
+        except:
+            from vllm.attention.backends.hpu_attn import (
+                HPUPagedAttentionMetadata as FlashAttentionMetadata)
+except (ModuleNotFoundError, ImportError, AssertionError) as err:
     raise RuntimeError(
         "Draft model speculative decoding currently only supports"
-        "CUDA and ROCm and HPU attention backend.")
+        "CUDA and ROCm and HPU attention backend.") from err
 
```

Comment:
@michalkuligowski , the previous CI failing is caused by ruff and yapf gave me conflicted fixing suggestion. So I changed the codes as above, which is acceptable to both yapf and ruff now.